integration/src/test/java/org/apache/iotdb/db/integration/IoTDBKeywordQueryIT.java

```diff
@@ -0,0 +1,68 @@
+package org.apache.iotdb.db.integration;
+
+import org.apache.iotdb.integration.env.EnvFactory;
+
+import org.junit.AfterClass;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+
+public class IoTDBKeywordQueryIT {
+
+  @BeforeClass
+  public static void setUp() throws Exception {
+    EnvFactory.getEnv().initBeforeClass();
+  }
+
+  @AfterClass
+  public static void tearDown() throws Exception {
+    EnvFactory.getEnv().cleanAfterClass();
+  }
+
+  @Test
+  public void testQuery() throws SQLException {
+    try (Connection conn = EnvFactory.getEnv().getConnection()) {
+      String[] sensorNames = new String[] {"select", "www.baidu.com"};
+      try (Statement stmt = conn.createStatement()) {
+        for (String sensor : sensorNames) {
+          stmt.execute("CREATE TIMESERIES root.sg.d1." + columnWrapper(sensor) + " int32");
+        }
+      }
+      try (PreparedStatement stmt =
+          conn.prepareStatement("INSERT INTO root.sg.d1(time, ?) values (?, ?)")) {
+        for (String sensor : sensorNames) {
+          stmt.setString(1, sensor);
+          stmt.setLong(2, 1L);
+          stmt.setInt(3, 1);
+        }
+      }
+      try (Statement stmt = conn.createStatement()) {
+        for (String sensor : sensorNames) {
+          try (ResultSet rs =
+              stmt.executeQuery("SELECT " + columnWrapper(sensor) + " FROM root.sg.d1")) {
+            Assert.assertTrue(rs.next());
+            long timestamp = rs.getLong("Time");
+            Assert.assertEquals(1L, timestamp);
+            int value = rs.getInt("root.sg.d1." + columnWrapper(sensor));
+            Assert.assertEquals(1, value);
+            Assert.assertFalse(rs.next());
+          }
+        }
+      }
+    }
+  }
+
+  /**
+   * columnWrapper is a special helper function to make business domain words compatible with IoTDB
+   * keywords
+   */
+  private String columnWrapper(String column) {
+    return String.format("`%s`", column);
```

Comment:
```suggestion
    return String.format("`%s`", column);
```
```suggestion
    if(column.contains("`")){
        column = column.replace("`", "``");
    }
    return String.format("`%s`", column);
```

Reply:
Thanks for the example~ Syntax convention related PR(https://github.com/apache/iotdb/pull/5696) has just been merged, now this demo could run as expected, I think. I will look into this issue futher once I finished tsfile module related work. Thanks again, enjoy your holiday~

Reply:
@Plutooooooo Thanks, but it seems that the error is still existed in the query related codes (`rs.getInt(String columnName)`):
1. If the column is wrapped by ``, then the query of 
```
rs.getInt("root.sg.d1.`select`")
```
throws NPE.
2. If not, then the query of 
```
rs.getInt("root.sg.d1.www.baidu.com")
```
throws NPE.

Now as a user I don't know which column I should use the wrapper.

Reply:
> @Plutooooooo Thanks, but it seems that the error is still existed in the query related codes (`rs.getInt(String columnName)`):
> 
> 1. If the column is wrapped by ``, then the query of
> 
> ```
> rs.getInt("root.sg.d1.`select`")
> ```
> 
> throws NPE. 2. If not, then the query of
> 
> ```
> rs.getInt("root.sg.d1.www.baidu.com")
> ```
> 
> throws NPE.
> 
> Now as a user I don't know which column I should use the wrapper.

Sorry I missed this part. It is confusing for the users indeed...... In SQL, when to use `` is specified, but in the header of result dataset, we only quote node that contains '`' or '.' which means node names that are keywords like "select" are not quoted. How to handle the header of result more properly is also an issue that needs further discussing.